### DISCUSSION TITLE
Welcome to Skills Network Author Discussions!

### DISCUSSION THREAD

**Discussion by reevejd** [ibm-skills-network/author-workbench-knowledgebase#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. This is the perfect place for you to:
- Get support with any issues you encounter while using Author Workbench. If you need help or encounter an issue while using Author Workbench, let us know by submitting a ticket and we'll get back to you!
- Suggest new features. If you see an idea you like, remember to upvote it or leave a comment. 👍
- Engage with other community members.
- Ask questions about Author Workbench

As a friendly reminder, please keep all discussions respectful and constructive. Let's maintain a welcoming and inclusive environment where everyone can feel comfortable sharing their thoughts and ideas. 🤝

Let's work together to make Author Workbench the best tool for content creators! 🚀

**Comment by aimfreedom** [comment]
I have a class about python on Coursera, here has a hand-on lab, but when I click the launch button, the author workbench website is showing page not found, how could I solve this issue? thanks.

**Reply by oliver-z-luo** [reply]
Hello @aimfreedom,
Sorry to hear you're running into issues with launching a lab, we are looking to this.

For us to help better, could you please answer the following questions:
1. Could you let me know the course you're trying to take? A link would be helpful
2. In terms of resolving the issue, could you let me know if you can access the page by refreshing (a few times)?

**Comment by JEALSON5** [comment]
reevejd. i am currently facing the same problem, i am unable to undertake by lab because the status of the jupyter is unkown and won't execute any thing i input

**Comment by dhruvg1505** [comment]
problem facing while launching the lab

**Comment by mi5aaaa** [comment]
Hello 

link https://author.skills.network/courses/7514 not opining and i need to take exam ASAP
